examples/mixedFF_complex/README.md

```diff
@@ -1,3 +1,3 @@
-## Combining a SMIRNOFF parameterized small molecule with an AMBER parameterized protein using ParmEd
+## `mixedFF_complex`: Combining a SMIRNOFF parameterized small molecule with an AMBER parameterized protein using ParmEd
```

Comment:
We don't need to include the name of the directory in any of these `README`s.

Reply:
Removed in eaaf7db